packages/utils/src/db/db.ts

```diff
@@ -219,6 +220,28 @@ export class HoprDB {
     await this.db.put(Buffer.from(this.keyOf(key)), Buffer.from(value))
   }
 
+  public dumpDatabase(destFile: string) {
+    log(`Dumping current database to ${destFile}`)
+    let dumpFile = fs.createWriteStream(destFile, { flags: 'a' })
+    this.db.createReadStream({ keys: true, keyAsBuffer: true, values: true, valueAsBuffer: true }).on('data', (d) => {
```

Comment:
Writing files usually requires *closing* the file after writing, so it could be worth to listen on the `end` event and then close the file.

Or, use the iterator support, e.g. 

```
for (const [key, value] of this.db.iterator) {
  ...
}
dumpFile.close()
```